### ISSUE TITLE
Need help for diagnostic performance bottleneck

### ISSUE THREAD

**Issue by vohai2003** [Engine-Simulator/engine-sim-community-edition#46]
Hi,
I've enjoyed this software for a while. But then I noticed, whenever I loaded run a heavy model, the sound and visual became glitchy, although neither CPU, GPU or memory reached limit.
Is there any possible way for me to diagnostic the bottleneck?

**Comment by Abdullah-Ajeebi** [comment]
Yes, you can, you just press N and scroll up or down for higher or lower
simulation frequencies respectively.

في الثلاثاء، ٣١ أكتوبر ٢٠٢٣, ٣:٢٠ ص vohai2003 ***@***.***>
كتب:

**Comment by vohai2003** [comment]
Hmm seem like the FPS also being affected, although (again) the performance limit isn't reached.
Lower the simulation frequency seemed to increase the FPS, and also eliminated the glitchy sound. But it may make problem when the engine running too fast.
So the problem is now migrated to diagnostic low FPS while performance limit is not reached.

**Comment by Abdullah-Ajeebi** [comment]
Hmm, that seems like the simulator is not using all cores.

في الثلاثاء، ٣١ أكتوبر ٢٠٢٣, ١١:١٢ ص vohai2003 ***@***.***>
كتب:

**Comment by vohai2003** [comment]
So there's no way to improve its performance on my side for now?

**Comment by Abdullah-Ajeebi** [comment]
Probably yes, but unless you know C++ and will to lose the ability to use
rotary engines, then yeah! You absolutely can.

في الثلاثاء، ٣١ أكتوبر ٢٠٢٣, ١١:٢٦ ص vohai2003 ***@***.***>
كتب:

**Comment by vohai2003** [comment]
Actually I'm not really familiar with C++ T_T

**Comment by Abdullah-Ajeebi** [comment]
Then... There's nothing you can do, unless you can spend some money on a
powerful computer with high single core performance.

في الخميس، ٢ نوفمبر ٢٠٢٣, ١:١١ م vohai2003 ***@***.***> كتب:
